### ISSUE TITLE
WCAG 4.1.2: ARIA hidden element must not contain focusable elements (th[aria-label="Answer"])

### ISSUE THREAD

**Issue by hash-logic** [hash-logic/issue#12]
**Issue**: `ARIA hidden element must not contain focusable elements` ([`aria-hidden-focus`](https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=msftAI))

**Target application**: [RFQ Edit · Custom Portal](https://mymatterssit.microsoftcrmportals.com/RFQ-Edit/?id=9805271b-8455-e911-b048-00155dd3f137)

**Element path**: th[aria-label="Answer"]

**Snippet**:

    <th style="width:13.88888888888889%;" class="sort-disabled" tabindex="0" aria-label="Answer" aria-hidden="true">Answer</th>

**How to fix**:

    Fix all of the following:
      Focusable content should have tabindex='-1' or be removed from the DOM

**Environment**:
Chrome version 73.0.3683.86

====

This accessibility issue was found using Accessibility Insights for Web 2.1.2 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
